receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -662,9 +662,15 @@ func TestScrapeMetrics_ProcessErrors(t *testing.T) {
 			expectedError: `error reading process name for pid 1: err1`,
 		},
 		{
-			name:          "Exe Error",
-			exeError:      errors.New("err1"),
-			expectedError: `error reading process executable for pid 1: err1`,
+			name:     "Exe Error",
+			exeError: errors.New("err1"),
+			expectedError: func() string {
+				if runtime.GOOS == "windows" {
+					return `error reading process executable for pid 1: err1; ` +
+						`error reading process name for pid 1: executable path is empty`
```

Comment:
Why it's triggered on Windows only?

Reply:
It's because windows implementation does not obtain a process name from some function call like on Linux and Darwin, it simply extracts it from the executable path. So on windows when the executable path is empty we expect an error with the message `executable path is empty` unlike some other errors returned from Linux or Darwin libraries.

Reply:
so basically when the executable path is empty on windows we return `fmt.Errorf("executable path is empty")` and on linux and darwin it is some error from psutil lib which we mock and that's why we expect other errors.

Reply:
Got it. Thanks for explaining. Please fix the rebase